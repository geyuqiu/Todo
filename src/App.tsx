import React from 'react';
import {Todos} from './component/Todos';

const App = () =>	<Todos/>;

export default App;
