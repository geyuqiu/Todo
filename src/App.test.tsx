import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

test('renders without crashing', () => {
    render(<App />);
    const title = screen.getByText(/Todos/i);
    expect(title).toBeInTheDocument();
});
