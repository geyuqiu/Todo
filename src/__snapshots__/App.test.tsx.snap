// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="ml-6 mr-3 sm:mx-8 pt-12"
  >
    <h1
      class="font-bold mb-3"
    >
      Todos
    </h1>
    <div>
      <input
        placeholder="keyword..."
        value=""
      />
    </div>
    <p
      data-testid="Table__Loading"
    >
      Loading ...
    </p>
  </div>
</DocumentFragment>
`;
